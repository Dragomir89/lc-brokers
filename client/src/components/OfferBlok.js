import React from 'react';
import { Link } from 'react-router-dom';

const OfferBlock = ({ address, price, area, floor, neighborhood, estateType, _id }) => {

    return (
        <div className='content-container__offer-wrapper'>
            <div >
                <img height='180px' src='https://t-ec.bstatic.com/images/hotel/max1024x768/137/137238081.jpg' />
            </div>
            <div style={
                {
                    marginLeft: '5px',
                    height: '180px',
                    position: 'relative'
                }

            }>
                <div className='test'>
                    <h3 className='title__offer'> {neighborhood} </h3>
                    <h3 className='title__offer'> {estateType} </h3>

                </div>

                <div className='offer-info-line'>
                    Адрес: {address} - етаж: {floor}
                </div>

                <div className='offer-info-line'>
                    Площ: {area}кв.
                    </div>
                <div>
                    Цена: {price} евро
                    </div>
                <div className='button__offer-details-position'>
                    <Link to={`/edit-offer/${_id}`}>
                        <button className='button__offer-details'>
                            Виж Детайли
                        </button>
                    </Link>
                </div>
            </div>
        </div>
    )
}

export default OfferBlock;
