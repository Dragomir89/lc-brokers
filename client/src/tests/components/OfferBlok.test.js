import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import '../setupTests';
import { offerBlockData } from '../test-data';
import OfferBlok from '../../components/OfferBlok';

test('should render OfferBlok correct', () => {
    const wrapper = shallow(<OfferBlok {...offerBlockData}/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
});