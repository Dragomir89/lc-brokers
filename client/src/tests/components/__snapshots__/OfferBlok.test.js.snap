// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render OfferBlok correct 1`] = `
<div
  className="content-container__offer-wrapper"
>
  <div>
    <img
      alt="estate"
      height="180px"
      src="https://t-ec.bstatic.com/images/hotel/max1024x768/137/137238081.jpg"
    />
  </div>
  <div
    style={
      Object {
        "height": "180px",
        "marginLeft": "5px",
        "position": "relative",
      }
    }
  >
    <div
      className="test"
    >
      <h3
        className="title__offer"
      >
         
        ЛОЗЕНЕЦ
         
      </h3>
      <h3
        className="title__offer"
      >
         
        ЧЕТЕРИСТАЕН
         
      </h3>
    </div>
    <div
      className="offer-info-line"
    >
      Адрес: 
      UPDATE_0001
       - етаж: 
      6
    </div>
    <div
      className="offer-info-line"
    >
      Площ: 
      40
      кв.
    </div>
    <div>
      Цена: 
      400
       евро
    </div>
    <div
      className="button__offer-details-position"
    >
      <Link
        to="/edit-offer/5bbcb3cc2a05d4701402868b"
      >
        <button
          className="button__offer-details"
        >
          Виж Детайли
        </button>
      </Link>
    </div>
  </div>
</div>
`;
